### PR TITLE
Verify empty response on normalizeExpandFalseForHybrid().

### PR DESCRIPTION
**Pull request by arlina-espinoza**
Fixing an issue related to #85 when the response is empty. Also added a test scenario for this.

### CODE REVIEW

File: src/Utility/ResponseToArrayHelper.php [hunk 1]
```diff
@@ -87,6 +87,11 @@ protected function responseToArray(ResponseInterface $response, bool $expandComp
      */
     protected function normalizeExpandFalseForHybrid(array $responseArray): array
     {
+        // If empty, no further processing is needed.
+        if (empty($responseArray)) {
+            return  $responseArray;
```
Comment (minnur) [inline src/Utility/ResponseToArrayHelper.php]: an extra space between return and the variable.

File: tests/Utility/ResponseToArrayHelperTest.php [hunk 2]
```diff
@@ -42,21 +42,39 @@ protected function setUp(): void
...
         $this->assertEquals($decodedEdgeStyle, $decodedWithCompatibility);
     }
+
+    /**
+     * DataProvider for testExpandCompatibility.
+     *
+     * @return array
+     *   The array of values to run tests on.
+     */
+    public function expandCompatibilityDataProvider() {
```
Comment (minnur) [inline tests/Utility/ResponseToArrayHelperTest.php]: Maybe this should be an array and converted to json via json_encode() so it is simpler to make changes ?

**Review by minnur — CHANGES_REQUESTED** [review]

File: src/Utility/ResponseToArrayHelper.php (hunk 1 above)
Comment (arlina-espinoza) [reply src/Utility/ResponseToArrayHelper.php]: Fixed!

File: tests/Utility/ResponseToArrayHelperTest.php (hunk 2 above)
Comment (arlina-espinoza) [reply tests/Utility/ResponseToArrayHelperTest.php]: Yeah, it would be clearer what the format of Hybrid responses are. Pushed the change. :+1:

**Review by minnur — APPROVED** [review]
Looks good to me.

**Pull request merged**